src/modules/TeamsList/components/Teams/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { PageTitle } from 'typography';
+
+import { TeamsHeader } from './components/TeamsHeader/TeamsHeader';
+import { MyTeam } from './components/MyTeam/MyTeam';
```

Comment:
change filename to index.tsx and import as import { MyTeam } from './components/MyTeam';

Reply:
done